config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -3,12 +3,15 @@ kind: OpenStackDataPlaneService
 metadata:
   name: nova
 spec:
-  secrets:
-    - nova-cell1-compute-config
-    # NOTE: this Secret needs to be created before deploying the data plane.
-    # It should contain an ssh key-pair in the secret fields: ssh-privatekey
-    # and ssh-publickey
-    - nova-migration-ssh-key
+  secrets: []
```

Comment:
im trying to add this to avoid 

 error: 'error in remove for path: ''/spec/secrets'': Unable to remove nonexistent key: secrets: missing value'

then we can remove this when we update the crd